src/main/java/entity/Post.java

```diff
@@ -0,0 +1,43 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Post extends ForumEntry {
+    private String postTitle;
+    private List<Comment> comments;
+    private String category;
+
+    public Post(String entryID, User author, Content content, LocalDateTime postedDate,
+                LocalDateTime editDate, int likes, int dislikes, String postTitle, 
+                List<Comment> comments, String category) {
+        super(entryID, author, content, postedDate, editDate, likes, dislikes);
+        this.postTitle = postTitle;
```

Comment:
Maybe throw an error if postTitle is null?

Reply:
Refer to the comment above